Sources/SwiftCurrent_SwiftUI/Views/WorkflowItem.swift

```diff
@@ -49,7 +49,9 @@ public struct WorkflowItem<F: FlowRepresentable & View, Wrapped: View, Content:
         ViewBuilder {
             if launchStyle == .navigationLink, let content = content {
                 content.navLink(to: nextView, isActive: $isActive)
-            } else if let body = model.body?.extractErasedView() as? Content, elementRef == nil || elementRef === model.body, launchStyle == .default {
+            } else if (wrapped as? WorkflowItemPresentable)?.workflowLaunchStyle == .modal, let content = content {
+                content.testableSheet(isPresented: $isActive) { nextView }
+            } else if let body = model.body?.extractErasedView() as? Content, elementRef == nil || elementRef === model.body, launchStyle != .navigationLink {
```

Comment:
I'm not entirely sure why `launchStyle != .navigationStyle`.  What scenario is that covering?

Reply:
That was the *first* thing we ran into. So the short version is that if that goes back to `launchStyle == .default` then the last screen in a series of modals (or the only screen if there is just one modal) is blank. Because the final screen just needs to render the view. 

So if FR1 -> FR2 (modal) then FR2 (modal) doesn't wrap itself in a sheet, it just *is*. Which is what that whole `if let body` line is about (just present myself).

Reply:
Ok.  So if we added other non-modal launch styles, we would need to be careful that this continues to work properly (alerts?).  But that's a future hypothetical problem to deal with.

Reply:
That's accurate but I might describe that in terms of how nav links work vs how modifiers work. Nav links take a destination and wrap a view to present. Modals, alerts, etc are *modifiers* that have access to the current view, have access to a binding boolean or item, and then wrap the thing they'll be presenting. 

So that's my long-winded way of saying that it's not non-modal launch styles, it's simply if we add anything else that works like nav links do, we'll need to modify this conditional again. But alerts would actually still work under this current logic.